src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/SwitchHelper.groovy

```diff
@@ -530,7 +534,17 @@ class SwitchHelper {
 
     @Memoized
     static boolean isVxlanEnabled(SwitchId switchId) {
-        return northbound.get().getSwitchProperties(switchId).supportedTransitEncapsulation
+        return getSwitchProperties(switchId).supportedTransitEncapsulation
                 .contains(FlowEncapsulationType.VXLAN.toString().toLowerCase())
     }
+
+    @Memoized
+    static List<SwitchPropertiesDto> getAllSwitchProperties() {
```

Comment:
This should be called `getAllSwitchPropertiesCached` or sth like that in order to clearly state that we are working with cache and not requesting fresh data from the server